airflow-core/src/airflow/utils/db_cleanup.py

```diff
@@ -523,3 +523,12 @@ def drop_archived_tables(
         session.execute(text(f"DROP TABLE {table_name}"))
         dropped_count += 1
     logger.info("Total dropped tables: %s", dropped_count)
+
+
+def calculate_table_size(
+    *,
+    table_name: str,
+    session: Session,
+) -> float | None:
+    """Calculate table size in megabytes."""
+    return 0
```

Comment:
Looks like the implementation is not done yet.

Reply:
Yeah. Wanted to keep few issues as good first issues for MLH internship folks to pick them up